src/jit/lsra.cpp

```diff
@@ -3614,21 +3461,24 @@ static int ComputeAvailableSrcCount(GenTree* node)
 }
 #endif // DEBUG
 
-static GenTree* GetFirstOperand(GenTree* node)
+TreeNodeInfo& LinearScan::GetTreeNodeInfo(GenTree* node, LocationInfoList& list)
```

Comment:
This looks like something that belongs in `LocationInfoList` class, not `LinearScan`.